### ISSUE TITLE
pytorch not uploading data to tensorboard page

### ISSUE THREAD

**Issue by enterthevoid22** [pytorch/pytorch#46202]
hello,

after spending days on this issue i'm trying again starting from scratch with a new environment. i'm going to put all details since i'm not formally educated with computers so maybe i make some dumb mistakes that i'm not aware of.

1. created a conda environment myenv with python 3.8, pytorch 1.6 and tensorboard 2.3.0

2. open pycharm, create new project, create new python module, named test.py
 

3. copy paste the code from tutorial in https://pytorch.org/docs/stable/tensorboard.html

also shown here
```
import torch
import torchvision
from torch.utils.tensorboard import SummaryWriter
from torchvision import datasets, transforms

# Writer will output to ./runs/ directory by default
writer = SummaryWriter()

transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.5,), (0.5,))])
trainset = datasets.MNIST('mnist_train', train=True, download=True, transform=transform)
trainloader = torch.utils.data.DataLoader(trainset, batch_size=64, shuffle=True)
model = torchvision.models.resnet50(False)
# Have ResNet model take in grayscale rather than RGB
model.conv1 = torch.nn.Conv2d(1, 64, kernel_size=7, stride=2, padding=3, bias=False)
images, labels = next(iter(trainloader))

grid = torchvision.utils.make_grid(images)
writer.add_image('images', grid, 0)
writer.add_graph(model, images)
writer.close()
```

4. run the code, check that folder was created and event files were generated. (folder path is C:\Users\david\PycharmProjects\TBtutorial\runs. inside that folder is another folder named after the moment it was generated and in it is one event file of type 0 file)

5. from windows toolbar enter anaconda prompt, activate myenv, change directory to /PycharmProjects/TBtutorial/runs.

6. type in: tensorboard --logdir==runs

7. copy paste link printed below on browser, get tensorboard blank page.

8. tried the same directly from pycharm terminal (typing tensorboard --logdir==runs ) instead of anaconda prompt

bottom line -
i always get the same result, the tensorboard web page without any image at the image thumbnail. i tried using python 3.6, tensorboard 1.15, pytorch 1.4 - all with one another. also tried to pass the writer other commands other than add_image, all to no use.

i'm clueless, will be grateful for any advice!

**Comment by vadimkantorov** [comment]
I don't know, but maybe it's again that it doesn't flush even if close is called

**Comment by enterthevoid22** [comment]
doesn't seem like the issue since i'm following the official tutorial on the pytorch website. 
still i tried adding flush but it didn't change anything.
is there a way to manually upload the event files to the web page? maybe it's possible to debug the issue this way.

**Comment by tigerccx** [comment]
I have a similar issue like this one. I am using tensoeboard 1.14.0, tensorboardX 2.1, pytroch 1.4.0+cu100 with python 3.7 on Windows 10. 
I can get the data to show in the tensorboard web page, the data will not be updated until I: 
1. shut down tensorboard in prompt with Ctrl+C. 
2. use tensorboard to open another folder (doesn't matter existing or not). 
3. refresh the webpage. 
4. shut down tensorboard again in prompt with Ctrl+C. 
5. use tensorboard to open the right folder. 
By this I can get the data to update on the web page. But this is not a long term solution. Do anyone have a solution for the issue?
